doc/source/workflows/basics.rst

```diff
@@ -446,7 +484,8 @@ Setting custom resources for tasks
 
 You can assign resources (e.g., CPUs, GPUs to tasks via the same ``num_cpus``, ``num_gpus``, and ``resources`` arguments that Ray tasks take):
 
-.. code-block:: python
+.. testcode::
+    :skipif: True
 
     @ray.remote(num_gpus=1)
```

Comment:
Is this skipped because of the `num_gpus=1`? If so, you could test this file with GPUs: https://docs.ray.io/en/master/ray-contribute/writing-code-snippets.html#how-to-test-examples-with-gpus

Reply:
(Might not be worth it just for this block, though)